src/go/k8s/tests/e2e-v2/upgrade-rollback/00-assert-create-redpanda.yaml

```diff
@@ -0,0 +1,41 @@
+---
+apiVersion: cluster.redpanda.com/v1alpha1
+kind: Redpanda
+metadata:
+  finalizers:
+    - operator.redpanda.com/finalizer
+  name: redpanda
+spec:
+  chartRef:
+    chartVersion: "4.0.13"
+  clusterSpec:
+    statefulset:
+      replicas: 1
+status:
+  conditions:
+    - message: Redpanda reconciliation succeeded
+      reason: RedpandaClusterDeployed
+      status: "True"
+      type: Ready
+  helmRelease: redpanda
+  helmReleaseReady: true
+  helmRepository: redpanda-repository
+  helmRepositoryReady: true
+---
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: redpanda
+spec:
+  replicas: 1
```

Comment:
I don't feel this is necessary in the assertion. Test should assert status part of the StatefulSet resource, right?